pkgs/by-name/li/libmamba/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  cmake,
+  fmt,
+  spdlog,
+  tl-expected,
+  nlohmann_json,
+  yaml-cpp,
+  simdjson,
+  reproc,
+  libsolv-with-conda,
+  curl,
+  libarchive,
+  zstd,
+  bzip2,
+  python3Packages,
+}:
+stdenv.mkDerivation rec {
+  pname = "libmamba";
+  version = "1.5.7";
+  src = fetchFromGitHub {
+    owner = "mamba-org";
+    repo = "mamba";
+    rev = "${pname}-${version}";
+    hash = "sha256-HfmvLi9IBWlaGAn2Ej4Bnm4b3l19jEXwNl5IUkdVxi0=";
+  };
+  nativeBuildInputs = [
+    cmake
+    python3Packages.python
```

Comment:
I specifically asked the author to use `python3Packages.python`.

The motivation is that we'd be forced to use `python3Packages` instead of `python3` if we later had to add some python packages as dependencies in order to keep splicing working and avoid bringing diverging versions of the interpreter into the closure. So, might as well always ask for `python3Packages` for consistency

Reply:
I suggest we merge this as is. If you want this changed to `python3` I suggest we open a separate PR and discuss it there

Reply:
If we you are worried about using different python package sets, then use python3.pkgs instead of python3Packages.

Reply:
No, `python3.pkgs` breaks splicing